composer.json

```diff
@@ -32,7 +32,9 @@
 		"nette/security": "^3.0",
 		"latte/latte": "^3.0",
 		"tracy/tracy": "^2.4",
-		"mockery/mockery": "^1.0"
+		"mockery/mockery": "^1.0",
+		"nette/coding-standard": "@dev",
+		"symplify/easy-coding-standard": "@dev"
```

Comment:
Have you tried installing "nette/coding-standard" as dependency? I think it should fail without this.